index.html

```diff
@@ -18,8 +18,22 @@
   </head>
```

Comment:
- [ ] [OPTIONAL] Kindly always consider adding more details to the PR description